mopidy_local/storage.py

```diff
@@ -120,12 +119,9 @@ def close(self):
     def clear(self):
         logger.info("Clearing image directory")
         try:
-            for root, dirs, files in os.walk(self._image_dir, topdown=False):
-                for name in dirs:
-                    os.rmdir(os.path.join(root, name))
-                for name in files:
-                    os.remove(os.path.join(root, name))
-        except Exception as e:
+            shutil.rmtree(self._image_dir)
+            self._image_dir.mkdir()
+        except IOError as e:
             logger.warning("Error clearing image directory: %s", e)
```

Comment:
f-strings while we are here?

Reply:
There's lots of modulo formatting in this file, so I opted to not change anything more than the lines I changed anyway.